Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.WebTester21/Pages/_TestInvokePanel.cshtml

```diff
@@ -1,63 +1,71 @@
-                         {
-                             <option value="@request.Filename">@request.Name</option>
-                         }
-                     </optgroup>
-                     }
-                 }
-             </select>
+
+
+    <div class="row">
+        <div class="col-sm form-group">
+            <label class="" for="sample-requests">Example Requests:</label>
+            <select class="control" id="sample-requests" onchange="onLambdaRequestSelected()">
+                <option id="void-select-request"> -- select a request -- </option>
+                @if (Model.SampleRequests.ContainsKey(SampleRequests.SampleRequestManager.SAVED_REQUEST_GROUP))
+                {
+                    <optgroup id="saved-select-request-group" label="@SampleRequests.SampleRequestManager.SAVED_REQUEST_GROUP">
+                        @foreach (var request in Model.SampleRequests[SampleRequests.SampleRequestManager.SAVED_REQUEST_GROUP])
+                        {
+                            <option value="@request.Filename">@request.Name</option>
+                        }
+                    </optgroup>
+                }
+                @foreach (var group in Model.SampleRequests.Keys)
+                {
+                    @if (!string.Equals(group, SampleRequests.SampleRequestManager.SAVED_REQUEST_GROUP))
+                    {
+                        <optgroup label="@group">
+                            @foreach (var request in Model.SampleRequests[group])
+                            {
+                                <option value="@request.Filename">@request.Name</option>
+                            }
+                        </optgroup>
+                    }
+                }
+            </select>
         </div>
-     </div>
+    </div>
 
-     <div class="row">
-         <div class="col-sm form-group">
-             <label class="col-xs-3 control-label" for="function-payload">Function Input:</label>
-             <textarea class="form-control" rows="10" id="function-payload" placeholder="JSON document as input to Lambda Function. Plain strings must be wrapped in quotes."></textarea>
-         </div>
-     </div>
+    <div class="row">
+        <div class="col-sm form-group">
+            <label class="col-xs-3 control-label" for="function-payload">Function Input:</label>
+            <textarea class="form-control" rows="10" id="function-payload" placeholder="JSON document as input to Lambda Function. Plain strings must be wrapped in quotes."></textarea>
+        </div>
+    </div>
 
-     <div class="row mt-3">
-         <div class="col-sm form-group">
-             <button class="btn btn-primary btn-sm" onclick="onExecute()">Execute Function</button>
-             <button type="button" class="btn btn-secondary btn-sm" data-toggle="modal" data-target="#save-request-prompt" onclick="onPreSaveClick()">Save Request</button>
-         </div>
-     </div>
+    <div class="row">
```

Comment:
Instead of making a the page extra long with this div what do you think about putting Function Input and Lambda Context into tabs with the Function Input as the first tab? That way the page is kept a bit simpler but the context tab is there for more advanced testing.